pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -423,6 +424,7 @@ func buildKubeApiserverConfig(
 	genericConfig.Authorizer = kubeAuthorizer         // this is used to fulfill the kube SAR endpoints
 	genericConfig.AdmissionControl = admissionControl
 	genericConfig.RequestContextMapper = requestContextMapper
+	genericConfig.DisabledPostStartHooks.Insert(rbacrest.PostStartHookName)
```

Comment:
https://github.com/kubernetes/kubernetes/pull/44633 has correct values for the RBAC resources.